README.md

```diff
@@ -71,6 +71,8 @@ Command Line Flag         | Description
 `-i`/`--include-linter`   | Specify which linters you specifically want to run
 `-x`/`--exclude-linter`   | Specify which linters you _don't_ want to run
 `-r`/`--reporter`         | Specify which reporter you want to use to generate the output
+`--fail-fast`             | Specify whether to fail after the first file with lint
+`--fail-level`            | Specify the severity above which the lint should fail
```

Comment:
This was a piece of documentation that we missed when we added the `fail-level` option. I saw it so added it before I'd forget.